cassiopeia/solver/SpectralNeighborJoiningSolver.py

```diff
@@ -0,0 +1,281 @@
+"""
+This file stores a subclass of DistanceSolver, SpectralNeighborJoining. This algorithm is based on the one developed by Jaffe, et al. (2020) in their paper titled "Spectral neighbor joining for reconstruction of latent tree models. 
```

Comment:
Please clean up formatting in this head docstring - you're missing an end quote after the title of the paper. Also, please add the journal name after the reference.

Reply:
Is there a specific format in which I should include the journal name or should I just say something like "in their paper... published in the SIAM Journal for Math and Data Science."? a brief google search didn't seem to give any standard formats for doing this in text.

(also I'm assuming this is intended but I'll mention it just in case it needs to be corrected, I just sorta copied the format of the blurb under NeighborJoining script and it doesn't mention a journal with the citation)